tests/test_snmpcommunity.rb

```diff
@@ -188,8 +255,18 @@ def test_snmpcommunity_group_set_valid
     # new group
     group = 'network-admin'
     snmpcommunity.group = group
-    assert_show_match(
-      pattern: /snmp-server community\s#{name}\sgroup\s#{group}/)
+    if platform != :ios_xr
```

Comment:
nit... this pattern is repeated ~5 times.  Consider creating a helper method for verifying the existence of a community by name via "show" output.